web_src/js/features/repo-settings.js

```diff
@@ -81,4 +82,28 @@ export function initRepoSettingBranches() {
     const $target = $($(this).attr('data-target'));
     if (this.checked) $target.addClass('disabled'); // only disable, do not auto enable
   });
+
+  // show the `Matched` mark for the status checks that match the pattern
+  const markMatchedStatusChecks = () => {
+    const patterns = document.getElementById('status_check_contexts')?.value.split(/[\r\n]+/);
+    const validPatterns = patterns.map((item) => item.trim()).filter((item) => item.length > 0);
```

Comment:
```suggestion
    const validPatterns = patterns.map((item) => item.trim()).filter(Boolean);
```

Intend is just to filter falsy values, right?

Reply:
done in 2f4c1a3